plaid/resources/plaid.toml

```diff
@@ -39,9 +39,9 @@ okta = 200
 [loading.memory_page_count.module_overrides]
 "example_rule.wasm" = 50
 
-[apis."okta"]
-token = ""
-domain = ""
+# [apis."okta"]
+# token = ""
+# domain = ""
```

Comment:
@obelisk After merging the change to support auth as an Okta app, this default config does not work anymore.

Reply:
Just delete it then :)